server/rest/analysis.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+from girder.api import access
+from girder.api.describe import Description
+from girder.api.rest import Resource
+
+from girder.plugins.minerva.utility.minerva_utility import findAnalysisFolder
+
+
+class Analysis(Resource):
+    def __init__(self):
+        self.resourceName = 'minerva_analysis'
+        self.route('GET', ('folder',), self.getAnalysisFolder)
+        self.route('POST', ('folder',), self.createAnalysisFolder)
+
+    @access.user
+    def getAnalysisFolder(self, params):
+        folder = findAnalysisFolder(self.getCurrentUser())
+        return {'folder': folder}
+    getAnalysisFolder.description = (
+        Description('Get the minerva analysis folder.'))
+
+    @access.user
+    def createAnalysisFolder(self, params):
+        folder = findAnalysisFolder(self.getCurrentUser(), create=True)
+        return {'folder': folder}
+    createAnalysisFolder.description = (
+        Description('Create the minerva analysis folder, a global resource.'))
```

Comment:
I guess we need to consider who will create the analysis folder and probably have some sort of group owner ship so everyone can read the items in it? This is related to how the analyses are imported as well.